independent-publisher-connectors/Safaricom Mpesa Daraja/apiDefinition.swagger.json

```diff
@@ -0,0 +1,696 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Daraja",
+    "description": "A connector that exposes daraja mpesa endpoints",
+    "version": "1.0",
+    "contact": {
+      "name": "John Muchiri, Bernard Karaba",
+      "url": "https://johnmuchirim.github.io/",
+      "email": "jeanfrancaiseharicot@gmail.com"
```

Comment:
@johnmuchiriM and @Bkaraba please notice that we have only one email address here. please confirm?